yb-voyager/cmd/importData.go

```diff
@@ -193,19 +193,48 @@ func getYBServers() []*tgtdb.Target {
 		log.Infof("Target DB nodes: %s", strings.Join(hostPorts, ","))
 	}
 
-	if parallelImportJobs == -1 {
-		parallelImportJobs = len(targets)
-	}
-
 	if loadBalancerUsed { // if load balancer is used no need to check direct connectivity
 		utils.PrintAndLog(LB_WARN_MSG)
+		parallelImportJobs = 2 * len(targets)
+		utils.PrintAndLog("Using %d parallel jobs by default. Use --parallel-jobs to specify a custom value", parallelImportJobs)
```

Comment:
After you address the comment for line 311, you won't need this additional `PrintAndLog()`.